src/index.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const state = {
+  tempCount: 49,
+  city: '✨✨✨',
+  sky: 'Sunny',
+};
+
+const convertKelvinToFarenheit = (kelvinTemp) => {
+  return Math.floor((kelvinTemp - 273.15) * (9 / 5) + 32);
+};
+
+const changeTempColor = () => {
+  let temp = state.tempCount;
+  if (temp <= 49) {
+    document.getElementById('tempCounter').style.color = 'teal';
+  } else if (temp <= 59) {
+    document.getElementById('tempCounter').style.color = 'green';
+  } else if (temp <= 69) {
+    document.getElementById('tempCounter').style.color = 'yellow';
+  } else if (temp <= 79) {
+    document.getElementById('tempCounter').style.color = 'orange';
+  } else {
+    document.getElementById('tempCounter').style.color = 'red';
+  }
+};
```

Comment:
Consider getting the element once at the start of the function and storing it in a variable so you don't need to repeatedly get it by ID.